### DISCUSSION TITLE
OLM update

### DISCUSSION THREAD

**Discussion by fgiloux** [operator-framework/operator-lifecycle-manager#2973] [Polls]
Today there is:
- no validation for OLM updates
- no documentation on how to update OLM
- no script similar to [install.sh](https://github.com/operator-framework/operator-lifecycle-manager/blob/master/deploy/upstream/quickstart/install.sh) for updates

[Background](https://github.com/operator-framework/operator-lifecycle-manager/issues/2969)

**Comment by coarasa** [comment]
I care about validation, but certainly having an upgrade path is a fundamental issue
